### DISCUSSION TITLE
Neutralino AppImage bundles

### DISCUSSION THREAD

**Discussion by krypt0nn** [neutralinojs/neutralinojs#755] [Show and tell]
I made npm / yarn package to bundle your project in AppImage: https://www.npmjs.com/package/neutralino-appimage-bundler

Probably will update it sometimes because use Neutralino in my project. You also can use it. Will be glad if it will help someone

**Comment by shalithasuranga** [comment]
Hey.. @krypt0nn Thank you very much for creating your project. Maybe I can provide some ideas to make your project more popular. neu CLI supports plugins. You can check more details here: https://github.com/neutralinojs/neutralinojs-cli#plugins
Sample plugin: https://github.com/neutralinojs/neutralinojs-cli-appify

I think your project is really helpful if neu developers can use with a command like `neu build-appimage`. Just an idea ;) Thanks again for the helpful project.
